virttest/utils_stress.py

```
@@ -197,3 +199,33 @@ def host_stress_event(self, event):
             else:
                 raise NotImplementedError
             time.sleep(self.itr_sleep_time)
+
+
+def install_stressapptest(vm):
+    """
+    Install stressapptest cmd
+
+    :param vm: the vm to be installed with stressapptest
+    """
+    session = vm.wait_for_login()
+    name = "git gcc gcc-c++ make"
+    try:
+        if not utils_package.package_install(name, session, timeout=300):
+            raise virt_vm.VMError("Installation of packages %s failed" %
+                                  name)
+    except Exception as detail:
+        raise detail
+
+    app_repo = "git clone https://github.com/stressapptest/" \
+               "stressapptest.git"
+    stressapptest_install_cmd = "rm -rf stressapptest " \
+                                "&& %s" \
+                                " && cd stressapptest " \
+                                "&& ./configure " \
+                                "&& make " \
+                                "&& make install" % app_repo
+    s, o = session.cmd_status_output(stressapptest_install_cmd)
+    if s:
+        logging.error("Failed to install stressapptest "
```

perhaps we can not allow silent failure,but throw exception here